Packages/Feature/Sources/Feature/Destinations.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  File.swift
+//  
+//
+//  Created by Eric Silverberg on 12/28/22.
+//
+
+import Foundation
+
+public enum Destinations {
```

Comment:
Maybe naming this enum singular (instead of plural) would eliminate any confusion when expecting parameters of this type:

```swift
// 👍 
func navigateTo(destination: Destination) { ... }

// ❓ 
func navigateTo(destination: Destinations) { ... } // it feels like we are going to navigate to multiple places
```